file-formats/sush/type/zif_aff_sush_v1.intf.abap

```diff
@@ -0,0 +1,186 @@
+INTERFACE zif_aff_sush_v1
+  PUBLIC .
+
+  types:
+    "! <p class="shorttext">Maintenance Mode</p>
+    "! Maintenance Mode
+    "! $values {@link zif_aff_sush_v1.data:co_usob_status_flag}
+    ty_usob_status_flag type c length 1,
+    "! <p class="shorttext">Maintenance Status</p>
+    "! Maintenance Status
+    "! $values {@link zif_aff_sush_v1.data:co_okflag}
+    "! $default {@link zif_aff_sush_v1.data:co_okflag.y_flag}
+    ty_okflag           type c length 1.
+
+  constants:
+    "! <p class="shorttext">Maintenance Mode</p>
+    "! Maintenance Mode
+    begin of co_usob_status_flag,
+      "! <p class="shorttext">Manual Maintenance</p>
+      "! Manual Maintenance
+      manual type ty_usob_status_flag value space,
+      "! <p class="shorttext">Autom. Maintenance (All Objects)</p>
+      "! Autom. Maintenance (All Objects)
+      a_flag type ty_usob_status_flag value 'A',
+      "! <p class="shorttext">Automatic Maintenance (Basis Authorization Objects Only)</p>
+      "! Automatic Maintenance (Basis Authorization Objects Only)
+      b_flag type ty_usob_status_flag value 'B',
+      "! <p class="shorttext">Application Does Not Require Default Values</p>
+      "! Application Does Not Require Default Values
+      i_flag type ty_usob_status_flag value 'I',
+      "! <p class="shorttext">Application is Deprecated</p>
+      "! Application is Deprecated
+      d_flag type ty_usob_status_flag value 'D',
+      "! <p class="shorttext">Application Is Obsolete</p>
+      "! Application Is Obsolete
+      o_flag type ty_usob_status_flag value 'O',
+    end of co_usob_status_flag,
+
+    "! <p class="shorttext">Maintenance Status</p>
+    "! Maintenance Status
+    begin of co_okflag,
+      "! <p class="shorttext">No Default</p>
+      "! No Default
+      x_flag type ty_okflag value 'X',
+      "! <p class="shorttext">Default With Field Values</p>
+      "! Default With Field Values
+      y_flag type ty_okflag value 'Y',
```

Comment:
You should rename the constants to something more meaningful. E.g., `default_with_values` or `default_with_field_values`.

```suggestion
      default_with_field_values type ty_okflag value 'Y',
```
This will lead to following syntax `"maintenanceStatus": "defaultWithFieldValues"` in the JSON file. If you kept it as specified above, it would be `"maintenanceStatus": "yFlag"`. If you keep the name of the field as `okflag` (see https://github.com/SAP/abap-file-formats/pull/467/files#r1064817735) it would be `"okflag": "yFlag"`.

From my point of view `"maintenanceStatus": "defaultWithFieldValues"` or `"maintenanceStatus": "defaultWithValues"`would be much more readable.

This is also relevant for all other constant values. ;)